lib/Windowing/README.MD

```diff
@@ -0,0 +1,124 @@
+# Windowing
+
+A selection of components to give you virtualisation.
+
+## How does it work?
+
+A `div` is rendered that is high enough to hold every item, it is simply calculated as follows:
+
+`listHeight = itemHeight * allWindowingIds`
+
+Each `item` has to have the same height, making all of this maths a lot easier.
+
+This `div` is wrapped in a container, with a set height, the `containerHeight`. This container is our window to look
+at the long list of items, it restricts the number of items we can see at a time though!
+
+When a scrolling event is triggered on the long list of items we retrieve a value called `scrollTop`, this represents
+the number of pixels from the top of the list that we have scrolled.
+
+With all of these values we can calculate which items can be seen through the window. Here is a quick example of the
+window being at the top of the list.
+
+#### Example 1 - Beginning of list
```

Comment:
It might be worth explaining that these examples are here to show the output data-wise? I mean, `usage` and stories show examples so I'm not too sure if these are readable to not.

Reply:
Maybe it's just the way it's written by you could write it up like;

- Let's go through an example...here are my properties "{ show object }"
- Ok, without any scrolling here are the items to show...here is why.
- Ok, now lets scroll...here's the new IDS

No need to go into the actually implementation detail like - "here is why - let's go over the math" but I feel like it could be explained a little better than

-------

With a fresh pair of eyes do these examples help more than stories?